File: leetcode/1119.py
"""
Runtime: 30 ms (beats 87.60%)
Memory: 13.8 MB (beats 51.74%)
"""


class Solution:
    def removeVowels(self, s: str) -> str:
        for vowel in ("a", "e", "i", "o", "u"):
            s = s.replace(vowel, "")
        return s

File: leetcode/1119.cpp
// Runtime: 2 ms, faster than 44.26% of C++ online submissions for Remove Vowels from a String.
// Memory Usage: 6.3 MB, less than 77.87% of C++ online submissions for Remove Vowels from a String.
class Solution {
 public:
  string removeVowels(string s) {
    string res = "";
    string vowels = "aeiou";
    for (auto c : s) {
      if (vowels.find(c) == string::npos) res.append(1, c);
    }

    return res;
  }
};
